os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -52,14 +52,15 @@
 		in->priv_len = len;                             \
 	} while (0)
 
-/* Max Key or Secure Storage Index */
+/* Max Factory Key and usable Key Index (Only allocate 8 slot) */
+#define FACTORY_KEY_INDEX_MAX 32
+#define USABLE_FACTORY_KEY_INDEX 8
+
+/* Max RAM Key and Total Key Index */
 #define KEY_STORAGE_INDEX_MAX 32
+#define TOTAL_KEY_STORAGE_INDEX_MAX FACTORY_KEY_INDEX_MAX + KEY_STORAGE_INDEX_MAX
```

Comment:
I think naming is complex, may be TOTAL_KEY_STORAGE_INDEX is fine?

Reply:
Done.